### DISCUSSION TITLE
Is there an option to close the modal bottom sheet when clicking outside of the sheet?

### DISCUSSION THREAD

**Discussion by Maurits97** [gorhom/react-native-bottom-sheet#651] [Q&A] [answered]
I was looking for a prop or something that could activate this behaviour, but I couldn't find it. Does somebody know how I can achieve this?

EDIT: I fixed it by adding a component with an onPress within my custom backdrop. Problem solved.

**Comment by gorhom** [comment, marked as answer]
you could look at https://gorhom.github.io/react-native-bottom-sheet/components/bottomsheetbackdrop 👍
